packages/dataviews/src/types.ts

```diff
@@ -247,6 +247,12 @@ export interface NormalizedFilter {
 	isPrimary: boolean;
 }
 
+export enum Density {
```

Comment:
Why is this not just `Density = 'compact' | 'medium' | 'comfortable'` ?

Reply:
It could be that. At first I did it because I didn't think the consumers should care about the values and used the enum. Since most cases would just handle this with css (like `is-density-${densityOption}`) classname though, I can change that, because consumers would need to add the classname and therefore know the value.